OverlayPlugin.Core/NetworkProcessors/LineCountdown.cs

```diff
@@ -0,0 +1,121 @@
+﻿using System;
+using System.Reflection;
+using System.Runtime.InteropServices;
+using RainbowMage.OverlayPlugin.MemoryProcessors;
+
+namespace RainbowMage.OverlayPlugin.NetworkProcessors
+{
+    [StructLayout(LayoutKind.Explicit, Size = structSize, Pack = 1)]
+    internal unsafe struct Countdown_v655
+    {
+        // 6.5.5 packet data (minus header):
+        // 34120010 4F00 1500 53  00  00  0102030405060708090A0B0C0D0E0F101112131415161718191A1B1C1D1E1F20 00000000
+        // AAAAAAAA CCCC BBBB DD  EE  FF  GGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGG HHHHHHHH
+        // 0x0      0x4  0x6  0x8 0x9 0xA 0xB
+        // Actor ID Wrld Time Res Unk Unk Name
+
+        public const int structSize = 48;
+        [FieldOffset(0x0)]
+        public uint actorID;
+        [FieldOffset(0x4)]
+        public ushort senderWorldId;
+
+        [FieldOffset(0x6)]
+        public ushort time;
+        [FieldOffset(0x8)]
+        public byte result;
```

Comment:
What does `result` represent?

Reply:
It's a result code that changes depending on whether starting the countdown was successful. e.g. trying to countdown in combat will give a different code, or trying to countdown when there's already a countdown active (by opening the GUI for the countdown, then someone else starting one, then you trying to start one).

You can see some of the various messages in `LogMessage.csv` from datamining, though there's not a direct mapping from the network packet to the actual EXD row that is displayed.

```
5255,57,33,0,False,"Battle commencing in <Value>IntegerParameter(1)</Value> <If(Equal(IntegerParameter(1),1))>second<Else/>seconds</If>!
5258,60,9,0,False,"Unable to initiate countdown while engaged in combat."
5259,60,9,0,False,"Unable to initiate countdown."
```

Reply:
In that case, I wonder if the various types of failures are more clutter than useful. I'm trying to imagine a use case where you'd want to know about a failure.

Reply:
From a log line perspective, you'd want to know success vs failure. And since we'd be writing something for the field regardless there, why not print the actual result code?

The general rule of thumb with these custom log lines is that if we get a packet, we write the line, even if it's not 100% useful for our known use cases, just for consistency's sake.

Reply:
I guess the root of my concern is that for the most common use case for these lines (knowing when combat is about to start, for timelines and such), you would want to have a definite "yes this is successful" flag. If 0 is the only code that indicates success, and non-zero always indicates a failure, then that's probably good enough.

Reply:
Yes, that's correct. `0` = success, non-`0` = failure. IIRC though it's been a month or two since I looked at the decompiled code that handles this, `result` is just a parameter passed to the handler func and the handler func does a switch/case or something over it to determine which sub-handler func to call.